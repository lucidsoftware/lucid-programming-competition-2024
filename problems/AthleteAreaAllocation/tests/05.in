version https://git-lfs.github.com/spec/v1
oid sha256:eb3b21584733edf6ab8bc58eb9da882d23e272efc0526ce3f6d722279827e5bd
size 7781
