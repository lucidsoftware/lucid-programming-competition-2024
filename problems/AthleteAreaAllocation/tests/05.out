version https://git-lfs.github.com/spec/v1
oid sha256:18c52269c5ca5001950d76d72d84d84f1985283dc4e19b8d659b57d2d27b157e
size 5
